src/Employer/Employer.Web/ViewModels/Part1/Wage/WageViewModel.cs

```diff
@@ -15,6 +15,14 @@ public class WageViewModel
 
         public string WageAdditionalInformation { get; set; }
 
+        public string MinimumWageStartFrom { get; set; }
+        public string NationalMinimumWageLowerBound { get; set; }
+        public string NationalMinimumWageUpperBound { get; set; }
+        public string NationalMinimumWageYearly { get; set; }
+        public string ApprenticeshipMinimumWage { get; set; }

```

Comment:
May be add Hourly to the name to make it obvious and consistent with "Yearly". Also same with lower and upper bound fields.

Reply:
👍 I'll do this in a future PR